src/System.Data.Common/System.Data.Common.sln

```diff
@@ -26,8 +26,8 @@ Global
 		Release|Any CPU = Release|Any CPU
 	EndGlobalSection
 	GlobalSection(ProjectConfigurationPlatforms) = postSolution
-		{B473F77D-4168-4123-932A-E88020B768FA}.Debug|Any CPU.ActiveCfg = netstandard-Debug|Any CPU
-		{B473F77D-4168-4123-932A-E88020B768FA}.Debug|Any CPU.Build.0 = netstandard-Debug|Any CPU
+		{B473F77D-4168-4123-932A-E88020B768FA}.Debug|Any CPU.ActiveCfg = netcoreapp-Debug|Any CPU
```

Comment:
This is done by vs2017. It changes this when you change the target build of a project in Build->Configuration manager. I had to do this, as the default was netstandard, so the code wouldn't  build (as the project system didn't honor the conditional include of the netcoreapp cs file, as the tooling for that is still being worked on (I use 15.2)